packages/mobile/src/transactions/feed/TransactionFeed.tsx

```diff
@@ -166,10 +166,6 @@ function TransactionFeed() {
       case 'TokenExchangeV2':
         return <ExchangeFeedItem key={tx.transactionHash} exchange={tx} />
       case 'TokenTransferV2':
-        if (!tokensInfo[tx.amount.tokenAddress]) {
```

Comment:
The blockchain-api should be the one that decides to show or not an item, not the client. If we haven't fetched the token list the feed shows up empty otherwise